### PR TITLE
Business client review backend

### PR DESCRIPTION
**Pull request by therealtgd**
Added backend service for posting reviews
Replaced all LocalDate.now().toEpochDay() and similar with DateTimeUtil.getTodayEpochMillisecond()

### CODE REVIEW

**Comment by iAmNikola** [comment]
lgtm

File: backend/src/main/java/team10/app/model/Review.java [hunk 1]
```diff
@@ -15,18 +16,28 @@ public class Review {
...
     private UUID id;
-    private String review;
+    private String message;
     private ReviewStatus status;
-    private LocalDate createdAt;
-    private LocalDate approvedAt;
-    private LocalDate answeredAt;
+    private long createdAt;
+    private long approvedAt;
+    private long answeredAt;
```
Comment (iAmNikola) [inline backend/src/main/java/team10/app/model/Review.java:23]: redundant properties. Should be deleted

File: backend/src/main/java/team10/app/util/Validator.java [hunk 2]
```diff
@@ -64,12 +66,19 @@ public class Validator {
...
+    // Account deletion request
     private static final int MIN_ACCOUNT_DELETION_REASON = 5;
     private static final int MAX_ACCOUNT_DELETION_REASON = 200;
 
+    // Reservation
     private static final int RESERVATION_MIN_PERSONS = 1;
     private static final int RESERVATION_MAX_PERSONS = 100;
 
+    // Review
+    private static final int REVIEW_MIN_MESSAGE = 3;
```
Comment (iAmNikola) [inline backend/src/main/java/team10/app/util/Validator.java:78]: This should be 5

**Review by iAmNikola — CHANGES_REQUESTED** [review]
all looks good. Only a couple changes needed

File: backend/src/main/java/team10/app/model/Review.java (hunk 1 above)
Comment (therealtgd) [reply backend/src/main/java/team10/app/model/Review.java:23]: Given that the functionality is not finished on the client side, I suggest we keep those properties.

**Pull request merged**